include/io/Display.hpp

```diff
@@ -1,4 +1,5 @@
 #include <cstdint>
+#include <stdexcept>
```

Comment:
My worry here is that std<array> doesn't crash the program when you try to access an out-of-range position. std::array<int,10> a; a[100] = 5 is undefined behavior, but, because std::array is implemented with a std::vector in our gcc compiler, the undefined behavior is just that the array gets resized and that the value is saved, no crash and no detection that something wrong is happening. I think you are right and maybe we should not worry, and that check is a too high to pay value, but I wanted to have a mechanism to know if a ROM is trying to save pixels that are wrong. Maybe we can just add this as an option?

Reply:
My worry here is that std doesn't crash the program when you try to access an out-of-range position. std::array<int,10> a; a[100] = 5 is undefined behavior, but, because std::array is implemented with a std::vector in our gcc compiler, the undefined behavior is just that the array gets resized and that the value is saved, no crash and no detection that something wrong is happening. I think you are right and maybe we should not worry, and that check is a too high to pay value, but I wanted to have a mechanism to know if a ROM is trying to save pixels that are wrong. Maybe we can just add this as an option?